website/source/intro/getting-started/build.html.md

```diff
@@ -74,6 +74,9 @@ AWS access key and secret key, available from
 We're hardcoding them for now, but will extract these into
 variables later in the getting started guide.
 
+~>**Note**: If you simply leave out AWS credentials, Terraform will automatically search for saved API credentials (for example, in ~/.aws/credentials) or IAM instance profile credentials. This option is much cleaner for situations where tf files are checked into source control or where there is more than one admin user.
+See details [here] (https://aws.amazon.com/blogs/apn/terraform-beyond-the-basics-with-aws/). Not only do you NOT want AIM credentials in source control, but you also want different IAM credentials for each admin user.
```

Comment:
This is a great point to add, but a couple suggestions:
- These lines aren't wrapped at 80 characters like the rest of the file.
- There is a typo on the second sentence of L78, `:%s/AIM/IAM/`.
- That last sentence could be touched up a bit to make it clearer. Perhaps something along the lines of: `Leaving IAM credentials out of the Terraform configs allows you to leave those credentials out of source control, and also use different IAM credentials for each user without having to modify the configuration files.`